src/Extensions/Hosting/Enviroments.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Omex.Extensions.Hosting
+{
+	/// <summary>
+	/// Provides variables with commonly used environments
+	/// </summary>
+	public static class OmexEnviroments
+	{
+		/// <summary>
+		/// Development environment, previously called Pr
+		/// </summary>
+		public static string Development { get; } = Environments.Development;
+
+
+		/// <summary>
+		/// Int environment
+		/// </summary>
```

Comment:
For those Ext to Microsoft, I think we should expand the name "Int" :)

Reply:
I don't know what does it mean :) We had an argument inside of a team about it top options was Integration and Internal, could you please provide clarity and I'll add into a comment?

Reply:
It looks like its Integration, after all, mystery solved, I've updated comments, thank you